content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst

```diff
@@ -0,0 +1,80 @@
+Marketing activities
+====================
+
+When creating a campaign in the :guilabel:`Marketing Automation` app, users can plan marketing
+activities such as email or SMS campaigns. To get started, navigate to the :guilabel:`Workflow`
+area, and click :guilabel:`Add New Activity`. A blank activity template will appear where
+parameters can be set for that specific activity.
+
+When :guilabel:`Add New Activity` is selected, a blank activity template will appear.
+
+.. image:: workflow_activities/activity-template.png
+   :align: center
+   :alt: An activity template in Odoo Marketing Automation.
+
+- :guilabel:`Activity Name`: the title of the activity.
+- :guilabel:`Activity Type`: choose between Email, Server Action (internal Odoo operation), or SMS.
+- :guilabel:`Mail Template`: choose from pre-configured templates (or create a new one on-the-fly).
+- :guilabel:`Trigger`: choose when this activity should be triggered.
+- :guilabel:`Expiry Duration`: configure to stop the actions after a specific amount of time
+  (after the scheduled date). When selected, a :guilabel:`Cancel after` field appears, in which the
+  user can choose how many :guilabel:`Hours, Days, Weeks, or Months` they want the actions to cease
+  after the initial date.
+- :guilabel:`Activity Filter`: domain related to this activity (and all subsequent Child Activities).
+- :guilabel:`Applied Filter`: activity will *only* be performed if it satisfies the specified
+  domain (filter).
+
+After the activity's settings are fully configured, click :guilabel:`Save & Close` to finish
+creating the activity.
+
+Workflow activity
+=================
+
+Once saved, the :guilabel:`Workflow Activity` will appear in Odoo. Each activity will be shown as a
+graph (by default). The configured trigger time for that activity can be found to the left of the
+:guilabel:`Workflow Activity` card.
+
+In the :guilabel:`Workflow Activity` section, there is also the number of activities that are
+successful (and rejected). The :guilabel:`Success` (green) and :guilabel:`Rejected` (red) numbers
+are color-coded in the graph for easy reference. 
+
+.. image:: workflow_activities/workflow-activity.png
```

Comment:
Optional: could probably bring this image down in size too. It's right on the cusp of being too large.